src/transport/RendezvousSessionDelegate.h

```diff
@@ -19,8 +19,11 @@
 
 #include <core/CHIPCore.h>
 #include <platform/internal/DeviceNetworkInfo.h>
+#include <transport/raw/MessageHeader.h>
+#include <transport/raw/PeerAddress.h>
 
 namespace chip {
+class TransportMgrBase;
```

Comment:
Please prefer includes to forward declarations (e.g. we include SystemPacketBuffer and we do not just forward declare Systel::PacketBuffer)